modules/ossm-configuring-jaeger.adoc

```diff
@@ -9,7 +9,7 @@ When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` resou
 
 The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as follows:
 
-.Default Jaeger parameters
+.Default "all-in-one" Jaeger parameters
```

Comment:
s/"all-in-one"/`all-in-one` ?

Reply:
Now that I'm looking at this again, since it's  a template, should I go  with `all-in-one`  ?   Can we use that formatting in headings?